src/System.Private.CoreLib/shared/System/Text/DecoderNLS.cs

```diff
@@ -391,7 +397,7 @@ internal int DrainLeftoverDataForGetChars(ReadOnlySpan<byte> bytes, Span<char> c
             // opportunity for any code before us to make forward progress, so we must fail immediately.
 
             _encoding.ThrowCharsOverflow(this, nothingDecoded: true);
-            throw null; // will never reach this point
+            throw null!; // will never reach this point
```

Comment:
Can you add a `// TODO-NULLABLE: https://github.com/dotnet/csharplang/issues/538` comment here?  This whole line shouldn't be necessary once we have that attribute and can attribute ThrowCharsOverflow appropriately.